documentation/src/main/asciidoc/topics/code_examples/RequestBalancing.java

```diff
@@ -0,0 +1,8 @@
+CacheContainer cacheContainer = new RemoteCacheManager();
+Cache<String, String> cache = cacheContainer.getCache();
+
+cache.put("key1", "aValue"); //this goes to s1
+cache.put("key2", "aValue"); //this goes to s2
+String value = cache.get("key1"); //this goes to s3
```

Comment:
It might be worth making it clear in the example that a failure occurs between the operations to cause it to go to the next node